activation_service_poc/docs/README.md

```diff
@@ -0,0 +1,93 @@
+# Node split PoC
+
+In our ongoing effort to lower the bar for smeshing, we're exploring a new direction for reducing system requirements for smeshers. The idea is to re-architect the internal modules in go-spacemesh, isolating the smeshing logic from the passive consensus code. By dividing the node along these lines, we can enable users to run the lightweight yet sensitive (requiring access to private key) smeshing logic separately from the rest of the node. This allows users with limited resources to use a remote node for their smeshing.
+
+We are demonstrating a PoC of the node split, where the smeshing logic is separated from the rest of the node. The PoC is based on the current go-spacemesh codebase and is intended to demonstrate the feasibility of the node split concept. The PoC is not intended to be a production-ready implementation.
+
+The architecture of the PoC is as follows:
+
+![design.png](design.png)
+
+The PoC consists of two separate processes:
+* `node service` aka node
+* `smesher service` aka client
+
+The easiest way to understand the setup is to try the PoC yourself. The following instructions will guide you through the process.
+
+
+## Running the PoC
+
+There are two distinct configuration/setup methods possible:
+1. Using locally running node and smesher service
+2. Using remote running node and local smesher service
+
+We will use a Docker Compose based setup for both options. While it's possible to run the PoC without Docker, it's more complicated, so for simplicity we'll use Docker.
+
+Both setups will run two smesher services connecting to the same node. Please note that one smesher service can have multiple post services connected to it, as is currently the case with node and post services. However, in this PoC we're using only one post service per smesher service.
```

Comment:
Actually the configuration for remote setup consist of 9 smesher services.

Reply:
Thanks, leftover from testing :)